content/blog/sivayanama-simplifying-open-api-specification-with-reusable-common-data-models.md

```diff
@@ -0,0 +1,114 @@
+---
+title: Simplifying OpenAPI specification with reusable common data models
+date: 2023-08-26T08:06:19.141Z
+author: BalaSubramanian Vetrivel
+authorimage: /img/vetrivel-balasubramanian1-photo.jpg
+disable: false
+tags:
+  - API, OpenAPI, OpenAPI specification, OpsRamp API, Yaml
+---
+The [OpenAPI specification](https://www.openapis.org)  is one of the most widely followed API contracts. It is language-agonistic.
+With the help of these OpenAPI specifications, clients can understand the API and invoke them without having access to the code or worrying about the implementation details. 
```

Comment:
Recommend modifying a bit here in these two sentences:
"The OpenAPI Specification (OAS) is one of..."
"With the help of the information found in the OpenAPI specification, clients can better understand APIs and how to invoke them..."